### PR TITLE
Added lambda handler function and additional formatting functions

### PR DESCRIPTION
**Pull request by hhm970**
- Created `load.py` as the pipeline's load script.
- Drafted out the lambda handler function to use in the `Dockerfile`.
- Due to complications in not formatting the `release_date` as a `datetime` object, functionality for this has been added in `load.py`.
- Created `sample_tags.py` purely as a means of visualising how distinct tags can be between websites.

### CODE REVIEW

**Review by Eibironke — APPROVED** [review]
Very descriptive docstrings, robust, great stuff 💯 💯 💯

File: pipeline/load/load.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+"""Given some input game data as a list[list] object, we load this data
```
Comment (shindym) [inline pipeline/load/load.py:1]: Good description

File: pipeline/load/load.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+def input_game_into_db(game_data: list[list], conn: connection) -> None:
+    """Given our game data, we insert each row into the game table, excluding
+    the tags and the platforms."""
+    with conn.cursor() as cur:
+        for game in game_data:
+            cur.execute(
+                """INSERT INTO game (name, description, price, developer, publisher, 
+                release_date, rating, website_id)
+                VALUES %s""", game[:-2])
```
Comment (shindym) [inline pipeline/load/load.py:48]: good use of formatting strings!

File: pipeline/load/load.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+def handler(event=None, context=None) -> None:
+    """Takes in an event (ie. the combined game data) and context, and
+    loads the game data into the database."""
+
+    event_data = event["game_data"]
+
+    if not event_data:
+        return None
+
+    load_dotenv()
```
Comment (shindym) [inline pipeline/load/load.py:118]: Not required
```suggestion
```

File: pipeline/load/load.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    conn = get_db_connection(ENV)
+
+    game_data = format_release_date_dt(event_data)
+
+    input_game_into_db(game_data, conn)
+
+    input_game_plat_into_db(game_data, conn)
+
+    input_game_tags_into_db(game_data, conn)
+
```
Comment (shindym) [inline pipeline/load/load.py:129]: close connection after

**Pull request merged**